docs/drivers/generic.md

```diff
@@ -33,25 +33,9 @@ to the host.
     $ docker-machine create \
       --driver generic \
       --generic-ip-address=203.0.113.81 \
-      --generic-ssh-key=~/.ssh/id_rsa \
+      --generic-ssh-key ~/.ssh/id_rsa \
       vm
 
-### Password-protected SSH keys
```

Comment:
Here I've removed the whole section as there is nothing special with password-protected keys. It had probably been added because of the bug described in  #3306.